codespell_lib/data/dictionary.txt

```diff
@@ -205,6 +222,7 @@ achor->anchor
 achored->anchored
 achoring->anchoring
 achors->anchors
+aci->acpi
```

Comment:
https://github.com/codespell-project/codespell/pull/567#pullrequestreview-137237332
(should we capitalize this?)

Reply:
done.